apps/maps/src/routes/+page.svelte

```diff
@@ -149,7 +149,7 @@
         return {
           html: `
             <h2 class="has-text-weight-bold has-text-teal-bold">
-              ${stop_name ?? '(Stop name unavailable)'}
+              ${stop_name ?? 'Non-stop segment'}
```

Comment:
Just noting that `non-stop` may be confusing like the adjective `nonstop`

Reply:
Right, though in this case I think either meaning is roughly correct (they're both not stops and the bus does not stop). These segments are created to provide more granular detail when there aren't any stops for long distances. "Stop name unavailable" implies a data issue which isn't the case.

Certainly open to other ideas if you have any though!

Reply:
I guess that makes sense, and no great alternatives come to mind.